### PR TITLE
[docs] Rewrite INSTALL.md

### PR DESCRIPTION
**Pull request by rtsisyk**
Add actual instructions for desktop/android/ios.

### CODE REVIEW

**Comment by rtsisyk** [comment]
@vng @int02h could you please validate updated instructions?

**Review by andrewshadura — CHANGES_REQUESTED** [review]
`Re-wrote` → `Rewrite` (commit summary needs to use imperative mood)

File: docs/INSTALL.md [hunk 1]
```diff
@@ -1,12 +1,81 @@
...
+- [iOS](#ios-app)
 
-    ./configure.sh
+## Desktop app
 
-## Desktop
+### Preparing
 
-You would need Cmake, Boost and Qt 5. With that, just run `tools/unix/build_omim.sh`.
+Get a Linux or a Mac machine.
```
Comment (andrewshadura) [inline docs/INSTALL.md:11]:
```suggestion
You need a Linux or a Mac machine to build OMaps.
```

File: docs/INSTALL.md [hunk 2]
```diff
@@ -28,174 +110,178 @@ and run
...
-    cmake <path_to_omim> -DCMAKE_BUILD_TYPE={Debug|Release}
-    make [<target>] -j <number_of_processes>
+## Android app
 
-It will compile binaries to the current directory. For a compiler, Clang
-is preferred, but GCC 6+ would also work.
+### Preparing
 
-### Ubuntu 20.04
+Get a Linux or a Mac machine.
```
Comment (andrewshadura) [inline docs/INSTALL.md:121]:
```suggestion
You need a Linux or a Mac machine to build OMaps for Android.
```

File: docs/INSTALL.md [hunk 3]
```diff
@@ -28,174 +110,178 @@ and run
...
+```
 
-## Designer Tool
+## iOS app
 
-### Building the tool
+### Preparing
 
-The designer tool has been merged into the master branch. You can build a package for macOS with:
+Get a Mac.
```
Comment (andrewshadura) [inline docs/INSTALL.md:213]:
```suggestion
Building OMaps for iOS requires a Mac.
```

File: docs/INSTALL.md [hunk 4]
```diff
@@ -28,174 +110,178 @@ and run
...
+ - Launch Terminal application on your Mac.
+ - Type `git` in the command line.
+ - Follow the instructions in GUI.
 
-* Build both the generator_tool and the designer tool
-* Generate data as usual (either with `generate_mwm.sh` or `generate_planet.sh`).
+Install [Xcode](https://apps.apple.com/ru/app/xcode/id497799835?mt=12) from AppStore.
 
-## Android
+Enroll [Apple Developer Program](https://developer.apple.com/programs/).
```
Comment (andrewshadura) [inline docs/INSTALL.md:225]:
```suggestion
Enroll in the [Apple Developer Program](https://developer.apple.com/programs/).
```

**Review by andrewshadura — CHANGES_REQUESTED** [review]

**Comment by rtsisyk** [comment]
Thanks, @andrewshadura! Have you tried to build the app by using these instructions?

**Comment by andrewshadura** [comment]
> ! Have you tried to build the app by using these instructions?

Not yet, but I certainly will.

**Review by andrewshadura — DISMISSED** [review]

**Comment by rtsisyk** [comment]
Need to mention how to install qt && cmake on Mac.

**Comment by rtsisyk** [comment]
@vng @biodranik please take a quick look. I try to actualize the build documentation based on my newbie experience.

File: docs/INSTALL.md [hunk 5]
```diff
@@ -28,174 +122,178 @@ and run
...
+Configure the repository for the debug mode:
+
+```bash
+./configure debug
+```
+
+Install [CocoaPods](https://cocoapods.org/):
+
+```bash
+sudo gem install cocoapods
```
Comment (vng) [inline docs/INSTALL.md:262]: I'd prefer here brew install cocoapods
sudo gem gives more troubles ..

**Review by vng — APPROVED** [review]

File: docs/INSTALL.md (hunk 5 above)
Comment (biodranik) [reply docs/INSTALL.md:262]: +1

**Comment by rtsisyk** [comment]
Updated according to comments.

**Review by vng — APPROVED** [review]

**Review by biodranik — APPROVED** [review]
LGTM

**Pull request merged**